nvflare/private/fed/simulator/simulator_server.py

```diff
@@ -144,6 +145,18 @@ def _create_server_engine(self, args, snapshot_persistor):
 
     def deploy(self, args, grpc_args=None, secure_train=False):
         super(FederatedServer, self).deploy(args, grpc_args, secure_train)
+        workspace = Workspace(args.workspace, "server", args.config_folder)
+        run_manager = RunManager(
+            server_name="server",
```

Comment:
following the other PR we will need to change this, can be done later.

Reply:
changed.